benchmarking/commons/benchmark_definitions/__init__.py

```diff
@@ -55,3 +55,35 @@
 from benchmarking.commons.benchmark_definitions.mlp_on_fashionmnist import (  # noqa: F401
     mlp_fashionmnist_benchmark,
 )
+from benchmarking.commons.benchmark_definitions.transformer_wikitext2 import (  # noqa: F401
+    transformer_wikitext2_benchmark,
+)
+
+__all__ = [
+    "SurrogateBenchmarkDefinition",
+    "RealBenchmarkDefinition",
+    "fcnet_benchmark_definitions",
+    "fcnet_benchmark",
+    "nas201_benchmark_definitions",
+    "nas201_benchmark",
+    "lcbench_benchmark_definitions",
+    "lcbench_selected_benchmark_definitions",
+    "lcbench_benchmark",
+    "yahpo_nb301_benchmark_definitions",
+    "yahpo_lcbench_benchmark_definitions",
+    "yahpo_lcbench_selected_benchmark_definitions",
+    "yahpo_iaml_benchmark_definitions",
+    "yahpo_iaml_selected_benchmark_definitions",
+    "yahpo_rbv2_benchmark_definitions",
+    "yahpo_rbv2_selected_benchmark_definitions",
+    "yahpo_nb301_benchmark",
+    "yahpo_lcbench_benchmark",
+    "yahpo_iaml_benchmark",
+    "yahpo_rbv2_benchmark",
+    "resnet_cifar10_benchmark",
+    "lstm_wikitext2_benchmark",
+    "real_benchmark_definitions",
+    "distilbert_imdb_benchmark",
+    "mlp_fashionmnist_benchmark",
+    "transformer_wikitext2_benchmark",
+]
```

Comment:
How did you discover that this was missing? Are there other `__init__.py` files where this might also be needed?

Reply:
This `__all__` is needed in order for links in the sphinx docs to work properly. I discovered it because links did not work. Let me check again over all `__init__.py`

Reply:
I added some more, now all `__init__.py` are fine